lib/liquid/standardfilters.rb

```diff
@@ -394,6 +394,12 @@ def default(input, default_value = ''.freeze)
 
     private
 
+    def get_property(input, property)
+      input[property]
+    rescue
```

Comment:
Is there a reason you changed this from explicitly rescuing `TypeError`?

Reply:
I think I just forgot. Thanks for catching that 👍